prow/gerrit/adapter/adapter.go

```diff
@@ -426,9 +422,6 @@ func (c *Controller) processChange(logger logrus.FieldLogger, instance string, c
 			}
 		}
 		if err != nil {
-			if setReviewWerr := c.gc.SetReview(instance, change.ID, change.CurrentRevision, failedIRCMsg, nil); setReviewWerr != nil {
```

Comment:
This codepath could be reached when there is anything else going on in the PR, for example a review comment message, which is not where this error message should be commented.

@mpherman2 , please take this in consideration when you work on this again